external/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/DockerJDBCIntegrationSuite.scala

```diff
@@ -99,6 +99,8 @@ abstract class DockerJDBCIntegrationSuite extends SharedSparkSession with Eventu
   val connectionTimeout = timeout(5.minutes)
   val keepContainer =
     sys.props.getOrElse("spark.test.docker.keepContainer", "false").toBoolean
+  val removePulledImage =
+    sys.props.getOrElse("spark.test.docker.removePulledImage", "true").toBoolean
```

Comment:
If this causes image downloading every time, this is a regression for the non-GA users, isn't it?
I'd keep the default as `false` and enable this `true` only at GA environment if that is trying to resolve GA issue.
> I investigated the reason and I found it's short of the storage capacity of the host on GA.

WDYT, @sarutak and @HyukjinKwon ?

Reply:
It was `false` at first but after [discussion](https://github.com/apache/spark/pull/32652#discussion_r639342223), I changed it to `true`.
non-GA users who have ever run docker integration tests can have images on their environment so image downloading doesn't happen every unless they remove images manually everytime.

Or, do you imagine a situation which is possible to cause a problem?

Reply:
To avoid behavior change, it would be better to keep it `false` but keeping container images in the local repository is side effect and done implicitly before this change. So I thought it is better do keep the images only if users can explicitly set the property to `false`.
Any thoughts? @dongjoon-hyun @HyukjinKwon

Reply:
Got it. If you think so, never mind. Since it's already merged, let's leave this AS-IS, @sarutak .

Reply:
I am okay either way. Please feel free to change if you guys have a preference 👍